pytorch_pipeline/components/trainer/component.py

```diff
@@ -26,47 +28,34 @@ def __init__(
         :param module_file_args : The arguments of the model class.
         :param trainer_args : These arguments are specific to the pytorch lightning trainer.
         """
-        if not module_file:
```

Comment:
@shrinath-suresh How is the actual input validations going to happen after the code structure change? In the new logic there is no validation check for the Trainer specific parameters.